packages/jest-util/src/buffered_console.js

```diff
@@ -15,6 +15,7 @@ import type {
   LogTimers,
 } from 'types/Console';
 
+import assert from 'assert';
```

Comment:
This is correct in node env, not in jsdom. Unsure if we care about that differentiation?

Reply:
The differences are small, but I'm not sure it should affect this fix

Reply:
Node:

![image](https://user-images.githubusercontent.com/1404810/36267282-cc2309b0-1273-11e8-97dc-bcb15b123133.png)

Chrome:
![image](https://user-images.githubusercontent.com/1404810/36267302-d8bc38fe-1273-11e8-9feb-765ef2d31875.png)

The output is pretty different

Reply:
@SimenB If I understand correctly, jsdom forwards its output to the Node.js console by default.

I'm not sure that we should print different messages (`AssertionError [ERR_ASSERTION]`/`Assertion failed`) depends on the environment.

What do you suggest?

Reply:
@cpojer thoughts?